pkg/execute/executor.go

```diff
@@ -53,6 +53,7 @@ const (
 	notifierStartMsg   = "Brace yourselves, notifications are coming from cluster '%s'."
 	notifierStopMsg    = "Sure! I won't send you notifications from cluster '%s' anymore."
 	unsupportedCmdMsg  = "Command not supported. Please run /botkubehelp to see supported commands."
+	restrictAccessMsg  = "Sorry, You can use botkube from configured channel only."
```

Comment:
```suggestion
	restrictAccessMsg  = "Sorry, You can use BotKube from configured channel only."
```

Reply:
This statement is not required now, so removed the message.